### PR TITLE
Implement bottom

### PR DESCRIPTION
**Pull request by DanielMorsing**
After the rework of the ordering for top, this was just a matter
of adding a flag telling the minheap which way the values should
be sorted.

### CODE REVIEW

File: tsdb/functions.go [hunk 1]
```diff
@@ -1407,7 +1422,7 @@ func ReduceTop(values []interface{}, c *influxql.Call) interface{} {
 		if whichselected == -1 {
 			// none of the points have any values
 			// so we can return what we have now
-			sort.Sort(topReduceOut{out})
+			sort.Sort(topReduceOut{out, c.Name == "bottom"})
```
Comment (corylanou) [inline tsdb/functions.go]: should `topReduceOut` -> `topBottomReduceOut`?

**Comment by corylanou** [comment]
One minor question, otherwise +1.  Looks fantastic!

File: tsdb/functions.go (hunk 1 above)
Comment (DanielMorsing) [inline tsdb/functions.go]: Missed that one.

**Comment by otoolep** [comment]
+1, but this does need a CHANGELOG entry under features.

**Pull request merged**